temba/flows/models.py

```diff
@@ -1665,10 +1672,13 @@ def start_msg_flow(self, all_contact_ids, started_flows=None, start_msg=None, ex
                 # check that we either have text or media, available for the base language
                 if (send_action.msg and send_action.msg.get(self.base_language)) or (send_action.media and send_action.media.get(self.base_language)):
 
+                    metadata = Flow.get_send_action_metadata(send_action)
+
                     broadcast = Broadcast.create(self.org, self.created_by, send_action.msg, [],
                                                  media=send_action.media,
                                                  base_language=self.base_language,
-                                                 send_all=send_action.send_all)
+                                                 send_all=send_action.send_all,
+                                                 metadata=metadata)
```

Comment:
Lets pass quick_replies down instead of all metadata, would rather this be explicit.

Reply:
If we work with all metadata, I think that in the future when we want to add one more key to metadata it would be easier, wouldn't be necessary change all send methods to do that for example.

Reply:
Think of metadata as being the storage mechanism for some of these different message options. The calling code shouldn't have to worry about how these things are stored - it just sends the message options it wants, and it's up to `Msg` to decide what is a field, what is embedded in metadata etc.

Reply:
Let's see if I understood, it should be applied just to Broadcast.send() method, the create method stores all metadata but send method only pass quick_replies, right?

Reply:
Internally in Msg and Broadcast we can pass around metadata, but calling code should be dealing with higher-level params like quick_replies